src/validateEmail.test.js

```diff
@@ -8,12 +8,71 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   it(`should return boolean`, () => {
+    expect(validateEmail('')).toEqual(expect.any(Boolean));
+  });
+
+  it(`should return 'true' for the valid email`, () => {
+    expect(validateEmail('adolas666@gmail.com')).toBe(true);
+  });
+
+  it(`should return 'true' for the email with integers`, () => {
+    expect(validateEmail('adolas666@gmail.com')).toBe(true);
+  });
```

Comment:
What is the difference between these tests?

Reply:
Repaired